poll/poll.py

```diff
@@ -501,7 +503,7 @@ def studio_submit(self, data, suffix=''):
 
         display_name = data.get('display_name', '').strip()
         if not question:
-            result['errors'].append("You must specify a question.")
+            result['errors'].append(self.ugettext("You must specify a question."))
             result['success'] = False
 
         answers = self.gather_items(data, result, 'Answer', 'answers')
```

Comment:
The third argument here and on the line below (`'Answer', 'Question') should be marked for translation. (But not the last argument, which is a python/json field ID)